CHANGES

```diff
@@ -4,6 +4,7 @@ Pint Changelog
 0.11 (unreleased)
 -----------------
 
+- Make `__str__` and `__format__` locale aware
```

Comment:
You say "locale aware" but it's not actually influenced by the locale environment by default.

Reply:
True, you can still use `set_fmt_locale("sys") which will use the system locale. The rationale behind being unaffected by the default locale (i.e. keeping it in the same language that the definition file) is to avoid affecting existing code.